pos_trade_receivable_autoreconcile/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# This file is part of OpenERP. The COPYRIGHT file at the top level of
+# this module contains the full copyright notices and license terms.
+
```

Comment:
Could you add a more classical header in each python file.
I know that licence is redundant in many python file but this is the general rules in OCA. And it allows to indicate differents authors if a file is written by many people.
sample : https://github.com/OCA/server-tools/blob/8.0/admin_technical_features/__openerp__.py